### PR TITLE
Image module ready

### PR DESCRIPTION
**Pull request by kikiondo**
This branch has one feature.

Show and add the image

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -3,8 +3,12 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '2.5.1'
 
+# shrine
```
Comment (diegosaul402) [inline Gemfile]: if you won't use this gem there is no need commit it

File: Gemfile [hunk 2]
```diff
@@ -3,8 +3,12 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
...
+
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '~> 5.2.1'
+
+gem 'pry'
```
Comment (diegosaul402) [inline Gemfile]: pry should be only on deb and test environments

File: app/controllers/users/confirmations_controller.rb [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class Users::ConfirmationsController < Devise::ConfirmationsController
+  # GET /resource/confirmation/new
```
Comment (diegosaul402) [inline app/controllers/users/confirmations_controller.rb:2]: You shouldn't commit commented code, better delete it or finish the implementation before ask for revision

**Review by diegosaul402 — CHANGES_REQUESTED** [review]
In general, looks good but have some issues to address in order to merge

File: Gemfile (hunk 2 above)
Comment (diegosaul402) [reply Gemfile]: Maybe I didn't explain correctly, you should move the `pry` gem to the test and development environments, and remove `byebug`, its not required if you're using pry

File: app/controllers/application_controller.rb [hunk 4]
```diff
@@ -1,2 +1,15 @@
 class ApplicationController < ActionController::Base
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+ protected
+
+ def configure_permitted_parameters
+   devise_parameter_sanitizer.permit(:sign_up, keys: [:nombre, :image])
```
Comment (diegosaul402) [inline app/controllers/application_controller.rb:7]: You're still using variable names in Spanish, change them.

**Review by diegosaul402 — CHANGES_REQUESTED** [review]
Attend this and previous comments before continue

**Review by archisss — APPROVED** [review]

**Comment by AlejandroHdzM** [comment]
LGTM

**Review by AlejandroHdzM — APPROVED** [review]
LGTM

**Pull request merged**